### PR TITLE
feat(frontend): add i18n namespace + remove setting help & id

### PR DESCRIPTION
**Pull request by marrouchi**
# Motivation

This PR : 
- We add namespace support to i18n
- Removed id + help from plugin settings
- Fix settings for input sort

# Type of change:

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by yassinedorbozgithub — APPROVED** [review]

File: frontend/public/locales/en/live-chat-tester.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "blue": "Blue",
+      "dark": "Dark"
+    },
+    "window_title": "Chat Window Title",
+    "avatar_url": "Chatbot Avatar URL",
+    "show_emoji": "Enable Emoji Picker",
+    "show_file": "Enable Attachment Uploader",
+    "show_location": "Enable Geolocation Share",
+    "allowed_upload_size": "Max Upload Size (in bytes)",
+    "allowed_upload_types": "Allowed Upload Mime Types (comma seperated)"
```
Comment (IkbelTalebHssan) [inline frontend/public/locales/en/live-chat-tester.json:26]:
```suggestion
    "allowed_upload_types": "Allowed Upload Mime Types (comma separated)"
```

File: frontend/public/locales/en/offline.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "blue": "Blue",
+      "dark": "Dark"
+    },
+    "window_title": "Chat Window Title",
+    "avatar_url": "Chatbot Avatar URL",
+    "show_emoji": "Enable Emoji Picker",
+    "show_file": "Enable Attachment Uploader",
+    "show_location": "Enable Geolocation Share",
+    "allowed_upload_size": "Max Upload Size (in bytes)",
+    "allowed_upload_types": "Allowed Upload Mime Types (comma seperated)"
```
Comment (IkbelTalebHssan) [inline frontend/public/locales/en/offline.json:26]:
```suggestion
    "allowed_upload_types": "Allowed Upload Mime Types (comma separated)"
```

File: frontend/public/locales/fr/contact.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "title": {
+    "contact": "Contact"
+  },
+  "label": {
+    "contact_email_recipient": "Email de contact du destinataire",
+    "company_name": "Le nom de votre entreprise",
+    "company_phone": "Tél.",
+    "company_email": "Adresse e-mail",
+    "company_address1": "Addresse",
+    "company_address2": "Addresse 2",
```
Comment (IkbelTalebHssan) [inline frontend/public/locales/fr/contact.json:11]:
```suggestion
    "company_address1": "Adresse",
    "company_address2": "Adresse 2",
```

**Review by IkbelTalebHssan — APPROVED** [review]
Hi @marrouchi ,
LGTM :rocket:  :tada:  just left some minor typo comments :pray: 
Thank you.

**Pull request merged**